paywall/tsconfig.json

```diff
@@ -19,7 +19,7 @@
     // "rootDir": "./",                       /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
     // "composite": true,                     /* Enable project compilation */
     "removeComments": false /* Do not emit comments to output. */,
-    //"noEmit": true,                           /* Do not emit outputs. */
+    "noEmit": true /* Do not emit outputs. */,
```

Comment:
```suggestion
    //"noEmit": true,                           /* Do not emit outputs. */
    "outDir": "/anything-other-than-.-will-work"
```

Reply:
So, if I comment it out again it fails obviously...

Reply:
and on all other settings the `,` is after the comment, so for consistency I'll use the same approach ;)

Reply:
did you try this fix?

Reply:
it succeeds here, so would be interesting if it doesn't there

Reply:
and to be clear: the original PR fails locally when I build with
<img width="1661" alt="Screen Shot 2019-06-25 at 12 19 14 PM" src="https://user-images.githubusercontent.com/98250/60115274-8ca90180-9743-11e9-9242-3d151371f88b.png">